### DISCUSSION TITLE
How to use the now() function inside the grid editor?

### DISCUSSION THREAD

**Discussion by ranme-papaya** [dbeaver/dbeaver#18665] [DBeaver Development]
Hi folks,

I'm wondering if there's an option to use the now() function inside the grid editor.

I'm using DBeaver and my connection is MariaDB.

I'm encountering the following error while trying to do so:

![NOW()](https://user-images.githubusercontent.com/118113921/210730581-b4a193f7-4e09-4ae8-834e-479e2bd3148f.png)

Is it even possible?

**Comment by LonwoLonwo** [comment]
Nope, only throw SQL Editor.

As a workaround for this function, you can use the following:

![2023-01-05 12_32_20-DBeaver Ultimate 22 3 1 - film](https://user-images.githubusercontent.com/45152336/210747799-7a8a73c0-61ea-4b2a-92f2-6de863021b2d.png)

**Reply by ranme-papaya** [reply]
Thank you very much! just what i needed!
